src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({ entries, onClick }) => {
+    const getChatLogJSX = (entries) => {
+        return entries.map((entry) => {
+            return (
+                <ChatEntry
+                    id={entry.id}
+                    key={entry.id}
+                    sender={entry.sender}
+                    body={entry.body}
+                    timeStamp={entry.timeStamp}
+                    liked={entry.liked}
+                    onClick={onClick}
+                />
+            );
+        });
+    };
+
+    return (
+        <div>
+            <ul className='chat-log'>{getChatLogJSX(entries)}</ul>
+        </div>
+    )
+}
+
+ChatLog.propTypes = {
+    entries: PropTypes.arrayOf(
+        PropTypes.shape({
```

Comment:
Nice use of PropTypes.shape for more detail on what data is expected for this component =]